js/classes/entities/midground/enemies/enemy_base.js

```diff
@@ -247,7 +252,10 @@ class EnemyBase {
             // Update bounding box and check collisions.
             this.enemy.lastBoundingBox = this.enemy.boundingBox;
             this.enemy.boundingBox = new BoundingBox(this.enemy.pos, this.enemy.scaledSize);
-            checkBlockCollisions(this.enemy, this.enemy.scaledSize);
+            const collisions = checkBlockCollisions(this.enemy, this.enemy.scaledSize);
```

Comment:
Thank you for implementing this! Very clean. Is collisions a good name? Just a thought, idk what it really is. Is it returning a boolean that says if an enemy has collided with a block?